app/src/main/java/org/bostwickenator/ftpuploader/MainActivity.java

```diff
@@ -136,9 +142,10 @@ public void run() {
                     }
                 });
 
-                String albumName = "FTP_Uploader_" + DateUtils.getDate();
-
-                ftp.createAndUseAlbumForSubsequentOperations(albumName);
+                if (settingsStore().getBoolean(SETTING_CREATE_ALBUM, true)) {
```

Comment:
Create a variable for the settings store, don't repeatedly call the instance for no good reason

Reply:
Compiler optimizes this and is much more readable. Just my opinion though. Against yours. ;-)